tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -323,6 +323,13 @@ tensorflow::Status ConvertGraphDefToTensorRT(
   return Status::OK();
 }
 
+struct EdgePtrCompare {
+  bool operator()(const tensorflow::Edge* lhs,
+                  const tensorflow::Edge* rhs) const {
+    return (lhs->id() < rhs->id());
```

Comment:
why do you prefer to use parenthesis for conditional expressions? Is that to protect against bugs caused by operator precedence rules?

Parenthesis usage is not very common and it is very safe otherwise, so we could drop them here for consistency.

Reply:
Ah I see, I thought I had seen it in the style reference but I think it might've been only in the case of multiple conditions/operator precedence.

Reply:
yes but only when there is an ambiguity based on the style in the rest of TensorFlow codebase, . For example, a mix of `and` and `or` conditions.

Reply:
I see, makes sense. Thanks!